### PR TITLE
Remove VR logo

### LINKED ISSUE #63: Remove VR logo
Since the portal will be "owned" by SciLifeLab, VR has asked us to remove their logo to keep in line with how they are represented elsewhere.

### PR DESCRIPTION
**Pull request by talavis**
Fixes #63

### CODE REVIEW

File: layouts/partials/footer.html [hunk 1]
```diff
@@ -68,7 +68,6 @@ <h4 class="small text-uppercase text-black-50">About</h4>
     <div class="col-md text-center"></div>
     <div class="col-md text-center"></div>
     <div class="col-md text-center my-3">
```
Comment (ewels) [inline layouts/partials/footer.html:70]:
```suggestion
    <div class="col-md"></div>
    <div class="col-md"></div>
    <div class="col-md"></div>
```

File: layouts/partials/footer.html [hunk 2]
```diff
@@ -68,7 +68,6 @@ <h4 class="small text-uppercase text-black-50">About</h4>
...
-      <a href="https://vr.se"><img src="/img/logos/vr-logo.png"></a>
     </div>
```
Comment (ewels) [inline layouts/partials/footer.html:71]:
```suggestion
```

Comment (talavis) [reply layouts/partials/footer.html:71]: Wouldn't this leave an empty tag?

Comment (talavis) [reply layouts/partials/footer.html:71]: Ah, other commit

**Pull request merged**